observability/src/main/java/io/grpc/observability/logging/Sink.java

```diff
@@ -29,6 +30,15 @@
    */
   void write(GrpcLogRecord message);
```

Comment:
This is only called from test code which means it is not used. Why is it there then?

Reply:
Actually if locationTags and customTags are set once in `GcpLogSink` so they don't need to be passed in each `write` then we just need this `write` and not the one below (line 38). Is that right?

Reply:
Yes, that is right. So passing locationTags and customTags to `GcpLogSink` constructor directly.